packages/terra-functional-testing/src/commands/expect/toMatchReference.js

```diff
@@ -35,8 +36,18 @@ function toMatchReference(screenshot) {
     message = message.concat(`Expected the screenshot to be within the mismatch tolerance, but received a mismatch difference of ${misMatchPercentage}%.`);
   }
 
+  let pass = global.Terra.serviceOptions.ignoreScreenshotMismatch || imagesMatch;
+
+  if (process.env.BUILD_TYPE === BUILD_TYPES.pullRequest && !pass) {
```

Comment:
We made change a while back on [PR #583](https://github.com/cerner/terra-toolkit/pull/583/files#diff-b5dcae7237ce9ea29604ce98c8e6e638fbafc997a6457e4d60630ccdbe7f47a2L14) to move any use of ENV variable to only be [read and set into the WDIO config options](https://github.com/cerner/terra-toolkit/pull/583/files#diff-b821c3a083e050c54034354280dac343f0b8d9ad66c460cad80a99ec7ebea640). Here is an [example PR](https://github.com/cerner/terra-toolkit/pull/646/files) where we created a new `USE_SELENIUM_STANDALONE_SERVICE` ENV in ion-jekins and being read and set into a CLI config option. 

There will be other places where we will need to read BUILD_TYPE so we should be setting this flag into `global.Terra.serviceOptions` where it can be accessed globally.

Reply:
Changed and fixed here https://github.com/cerner/terra-toolkit/pull/736/commits/e3ea3399352b384b0e54bca1fecb239726424110#diff-b821c3a083e050c54034354280dac343f0b8d9ad66c460cad80a99ec7ebea640R43